baseframe/forms/auto.py

```diff
@@ -26,25 +26,37 @@ def render_form(form, title, message='', formid='form', submit=__(u"Submit"), ca
     else:
         code = 200
     if request.is_xhr and ajax:
-        return make_response(render_template('baseframe/ajaxform.html', form=form, title=title,
+        template = 'baseframe/bootstrap3/ajaxform.html.jinja2'
+        if current_app.config['theme'] == "mui":
+            template = 'baseframe/mui/ajaxform.html.jinja2'
```

Comment:
Make this entire `if` condition simpler by doing this:
```python
template = THEME_FILES[current_app.config['theme']]['ajaxform.html.jinja2']
```

Reply:
Fixed